dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedMetricsTRSOpenApiIT.java

```diff
@@ -79,12 +79,16 @@
 import io.dockstore.webservice.core.metrics.ExecutionTimeStatisticMetric;
 import io.dockstore.webservice.core.metrics.MemoryStatisticMetric;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.time.Instant;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+
 import org.apache.http.HttpStatus;
+import org.jline.utils.Log;
```

Comment:
For the logger, we usually use `org.slf4j.Logger;`. An example declaration is https://github.com/dockstore/dockstore/blob/ce1911f8e5d0a9bc7713100eff39a97e9951f6a9/dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceFileHelper.java#L32